docs/reference/rest_api_communities.md

```diff
@@ -602,6 +602,151 @@ DELETE api/communities/<com_id>/logo HTTP/1.1
 HTTP/1.1 204 No Content
 ```
 
+### Get Community Members
```

Comment:
This one is documented here: https://inveniordm.docs.cern.ch/reference/rest_api_members/ .

There is something to be said about perhaps nesting it under a "Communities" subsection. I don't know if the doc software allows it.

Reply:
My vote would then be to remove that other documentation (or sub-split eventually).